modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityUserManager.cs

```diff
@@ -242,5 +243,12 @@ public virtual async Task<IdentityResult> AddDefaultRolesAsync([NotNull] Identit
 
             return await UpdateUserAsync(user);
         }
+
+        public virtual async Task ValidatePasswordAsync(Guid id, [NotNull] string username, [NotNull] string email,
+            string password, Guid? tenantId = null)
+        {
+            var user = new IdentityUser(id, username, email, tenantId);
+            (await base.ValidatePasswordAsync(user, password)).CheckErrors();
+        }
```

Comment:
If there is a password validator. It requires that the password cannot be the same as the user's first or last name or other properties.

A non-existent user can cause problems. Maybe we can consider ignoring the password validator for **admin**.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.usermanager-1.createasync?view=aspnetcore-5.0#Microsoft_AspNetCore_Identity_UserManager_1_CreateAsync__0_